src/containers/BrandsForm/BrandsForm.js

```diff
@@ -1,19 +1,80 @@
-import React from "react";
-
+import React, { useState } from "react";
 import { Button, TextField } from "@material-ui/core";
-
 import "./style.css";
 
-const BrandsForm = () => (
-  <section>
-    <form className="brandsForm" noValidate autoComplete="off">
-      <div className="brandsForm--inputs">
-        <TextField id="standard-basic" label="Marca" />
-      </div>
-      ß<Button variant="contained">Cadastrar</Button>
-      <Button variant="contained">Cancelar</Button>
-    </form>
-  </section>
-);
+const BrandsForm = ({ sendBrand }) => {
+  const [brand, setBrand] = useState("");
+  const [errors, setErrors] = useState({
+    validator: { isvalid: true, text: "" }
+  });
+
+  function validate(brand) {
+    let isBrandValid = {
+      isvalid: true,
+      text: "",
+    };
+
+    if (brand.length === 0) {
+      isBrandValid = {
+        isvalid: false,
+        text: "Preencha o campo Marca",
+      };
+    } else if (brand.length <= 2) {
+      isBrandValid = {
+        isvalid: false,
+        text: "A marca deve ter pelo menos 3 dígitos",
+      };
```

Comment:
E se a gente parasse de usar `else if` ou `else` nas nossas condicionais? Acho que essa partezinha poderia ser trocada por mais um if, já que o parametro vai passar por ela de qlq maneira. Daí ficaria mais legível também.
```suggestion
    } 
    if (brand.length <= 2) {
      isBrandValid = {
        isvalid: false,
        text: "A marca deve ter pelo menos 3 dígitos",
      };
```

Refs:
https://morioh.com/p/1c722b9742ac
https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
Hellou @miglsoares, acho uma ótima ideia, mas nesse caso se eu removo, ele cai na segunda quando ele nao preenche a marca, porque é o ultimo if que cai. 
Vou alterar também a posiçao dos ifs pra fazer isso!
Bju